packages/owl-bot/src/config-files.ts

```diff
@@ -0,0 +1,41 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// The .github/.OwlBot.lock.yaml is stored on each repository that OwlBot
+// is configured for, and indicates the docker container that should be run
+// for post processing:
+export interface OwlBotLock {
+  docker: {
+    image: string;
+    digest: string;
+  };
+}
+
+// The default path where .OwlBot.lock.yaml is expected to be found.
+export const owlBotLockPath = '.github/.OwlBot.lock.yaml';
+
+// Throws an exception if the object does not have the necessary structure.
+// Otherwise, returns the same object as an OwlBotLock.
+export function owlBotLockFrom(o: any): OwlBotLock {
```

Comment:
Do we really need to accept `any` here?  The typing has no runtime behavior changes in TypeScript, so unless you're passing in arbitrary objects intentionally from other parts of the code, using `OwlBotLock` here should be ok.  

Side note - please treat all lint warnings as errors.  We need to fix this in `gts` to report them as errors, but if we use `any` anywhere in the code base, it needs an eslint exception (and a good reason usually)

Reply:
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/js-yaml/index.d.ts#L12

js-yaml seems to defines its type as:

`object | string | number | null | undefined`

So perhaps we could use this rather than `any`?

Reply:
I removed the `any`.  I'd like to keep this file intentionally ignorant of yaml types.